status-im/status-react
# Jc/1 profile

fixes: https://github.com/status-im/status-mobile/issues/16067
This pr introduces a few small changes related to the create-profile page in onboarding flow:

- removes b & w option from color picker
|- additionally adds a unit test to color picker 

- adds neutral and positive message to user that name should be minimum 5 characters ( as in updated designs)
|- turns green and positive icon when username is a valid length ( and there is no validation errors)

- adjusts border color around the continue button to what is agreed in figma. There is some issues with aligning blur colors as they are shown in Figma. (After discussing with @smohamedjavid, we agreed to follow up with a discussion to the design team to create some common configurations in our codebase which will match the blur values they have set in Figma)

Pixel 7a
<img src="https://github.com/status-im/status-mobile/assets/22799766/57b23263-dfd2-4953-852f-4206047df7d9" width="310px" />

iPhone 12
<img src="https://github.com/status-im/status-mobile/assets/22799766/7bb21d5f-e229-4199-8dbc-9e756fc305c7" width="310px" />
To Test:
see the explanation above and review the Create Profile page in onboarding flow.
